update_feed.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 set -euo pipefail
+POSTGRES_DSN="$(python -c 'import configparser; c = configparser.ConfigParser(); c.read("config.ini"); print(c["gino"]["dsn"])')"
```

Comment:
might be better to put this as `python3`, since this is a python 3 only app, and some modern distros don't have `/usr/bin/python` by default. But other than that it looks fine